LeaderboardBackend/Models/Entities/User.cs

```diff
@@ -14,7 +14,7 @@ public class User
 	/// The User's name. Must be:
 	/// <ul>
 	///   <li>between 2 - 25 characters inclusive; and</li>
-	///   <li>a sequence of letters, each separated by zero or one of:</li>
+	///   <li>a sequence of letters or numbers, each separated by zero or one of:</li>
```

Comment:
nit: "An alphanumeric sequence" maybe? I don't think anyone's really gonna be that confused by this wording though

Reply:
```suggestion
	///   <li>an alphanumeric sequence, each separated by zero or one of:</li>
```